.github/workflows/macos-tests.yml

```
@@ -19,12 +19,12 @@ jobs:
     strategy:
       fail-fast: false 
       matrix:
-        PYTHON_VERSION: ['3.6', '3.7', '3.8']
+        PYTHON_VERSION: ['3.6', '3.7', '3.8', '3.9']
     steps:
       - name: Checkout branch
         uses: actions/checkout@v2
       - name: Install Conda
-        uses: goanpeca/setup-miniconda@v1
+        uses: conda-incubator/setup-miniconda@v2
         with:
            activate-environment: test
            auto-update-conda: true
```

We need to add here the config for the channel priority. It should be something like:

```suggestion
           auto-update-conda: true
           channels: conda-forge
           channel-priority: strict
```